src/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,602 @@
+// Navigation ------------- //
+
+$height-nav-secondary: 5rem;
+$width-nav-sidebar:    250px;
+
+
+.usa-header {
+  width: 100%;
+  z-index: 1;
+
+  @include media($nav-width) {
+    border-bottom: 1px solid $color-gray-light;
+  }
+
+  a {
+    border-bottom: none;
+  }
+
+  em {
+    display: block;
+    font-size: $h5-font-size;
+    font-style: normal;
+    font-weight: $font-bold;
+    margin: 0;
+
+    @include media($nav-width) {
+      display: block;
+      font-size: 2.4rem;
+      line-height: $heading-line-height;
+    }
+  }
+
+  .usa-search {
+    margin-bottom: 1.5rem;
+
+    @include media($nav-width) {
+      float: right;
+      margin-bottom: 0;
+      max-width: 21.5rem;
+    }
+  }
+}
+
+.usa-navbar {
```

Comment:
I think what Marco is saying is that we might want to start thinking about separating the header specific styles from the nav styles. This might help in creating the ability to using each of these sets of styles are their own. For example, we could possibly run into issues when want to use nav specific styles in another component, but are required to either remove or incorporate header styles.